docs/examples/KModal/KModal.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+
+  <div>
+    <KButton @click="openModal">Click Me</KButton>
+    <KModal
+      v-if="showModal"
+      :appendToOverlay="true"
+      title="Dialog Box"
+    >
+      <template> Dialog Box with description </template>
```

Comment:
```suggestion
      <template> Dialog box with description </template>
```

Aligns case with writing guidelines.